src/backend-c/runtime/Makefile

```diff
@@ -15,19 +15,18 @@ boehm-gc-install :
 %.c :
 
 %.o : %.c
-	cc $(NDEBUG) -std=c99 $< -c -o $@
+	cc $(NDEBUG) -Iboehm-gc-install/include -std=c99 $< -c -o $@
 
-runtime.o : io.o assoc_stack.o hashcons.o
+runtime.o : boehm-gc-install/lib/libgc.a io.o assoc_stack.o hashcons.o cast_queue.o
```

Comment:
Should you add `boehm-gc-install/lib/libgc.a` to the recipe boehm-gc-install above? Something like this?
```
boehm-gc-install boehm-gc-install/lib/libgc.a:
```

Reply:
`boehm-gc-install/lib/libgc.a` subsumes `boehm-gc-install` so I replaced the latter with the former. I also added a recipe with short name that requires this library that and easy to use in other recipes.